src/util/getTopologicalOrder.ts

```diff
@@ -3,32 +3,36 @@ export interface Edge {
 	target: string;
 }
 
-// Uses the Khan Algorithm loosely
+/**
+	* Uses the Khan Algorithm (this implementation cannot work with isolated nodes)
+	* @param {Set<Edge>} graph - A graph represented as set of edges
+	* @return {Array<string>} The topologial oder of the graph
```

Comment:
```suggestion
	* @return {Array<string>} A topological ordering of the graph
```
There could be multiple such orderings after all.